### ISSUE TITLE
Background Color Not Displayed for ListTile and Container Inside Row or Column

### ISSUE THREAD

**Issue by perna-python** [flet-dev/flet#3426]
**Description**
When inserting a ListTile or a Container inside a Row or Column and defining a bgcolor, the background color is not displayed. However, it works correctly outside of these widgets. Another issue occurs when specifying the bgcolor of a ListTile and the bgcolor of the Container in which it is placed; only the bgcolor of the Container is displayed.

Below are the examples:

**Code example to reproduce the issue:**

```python
import flet as ft

def main(page:ft.Page):
    lv = ft.ListView(spacing=10)

    for i in range(0,5):
        lt = ft.ListTile(title=ft.Text(i), bgcolor=ft.colors.GREEN, on_click=lambda e: print(e.control.bgcolor))
        lv.controls.append(lt)

    page.add(
        ft.Row(
            controls=[
                lv 
            ]
        )
    )

if __name__ == "__main__":
    ft.app(target=main)
```

```python
import flet as ft

def main(page:ft.Page):
    lv = ft.ListView(spacing=10)

    for i in range(0,5):
        lt = ft.ListTile(title=ft.Text(i), bgcolor=ft.colors.GREEN, on_click=lambda e: print(e.control.bgcolor))
        lv.controls.append(lt)

    page.add(
            ft.Container(
            content=lv,
            bgcolor=ft.colors.RED
        ))

if __name__ == "__main__":
    ft.app(target=main)
```

**Describe the results you received:**

Example 1:
![image](https://github.com/flet-dev/flet/assets/56712747/95b255ab-43c7-4be6-9365-56dbcfe08d3f)

Example 2:
![image](https://github.com/flet-dev/flet/assets/56712747/38adccab-93d3-4654-a223-9229e2ca9a85)

**Describe the results you expected:**

In the first case, I expect the bgcolor of the ListTile to be displayed. In the second case, I expect both the bgcolor of the ListTile and the bgcolor of the Container to be displayed.

**Additional information you deem important (e.g. issue happens only occasionally):**

**Flet version (`pip show flet`)**:

```
Name: flet
Version: 0.22.1
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page:
Author: Appveyor Systems Inc.
Author-email: hello@flet.dev
License: Apache-2.0
Requires: cookiecutter, fastapi, flet-runtime, packaging, qrcode, uvicorn, watchdog
Required-by:
```

**Give your `requirements.txt` file (don't `pip freeze`, instead give direct packages):**

```
flet
```

**Operating system**:

Windows 10, macOS,

**Additional environment details:**

**Comment by ndonkoHenri** [comment]
For some context, here is the Flutter error message:
```
Vertical viewport was given unbounded width.
Viewports expand in the cross axis to fill their container and constrain their children to match
their extent in the cross axis. In this case, a vertical viewport was given an unlimited amount of
horizontal space in which to expand.
```

To solve this, you need to expand the `ListView`, or give it a width constraint (not equal to infinity):
```python
lv = ft.ListView(spacing=10, expand=True)
# or 
lv = ft.ListView(spacing=10, width=500)
```
